llama/convert.py

```diff
@@ -134,5 +135,5 @@ def tiny_llama(model_path):
     weights, params = globals()[args.model_name](model_path)
     np.savez(str(model_path / "weights.npz"), **weights)
     if params is not None:
-        with open(model_path / "params.json", "w") as fid:
+        with open(model_path / "config.json", "w") as fid:
```

Comment:
@pcuenca sorry this example is a bit of a mess (trying to keep support for everything).

The Meta distributed llama models come with `params.json`. So if you load them in the `llama` convert (around L47) and just return them instead of returning `None`, that should work nicely

Reply:
My bad, I read the code too fast and now I see I only updated the `tiny_llama` case.